learning_wavelets/evaluation_scripts/exact_recon_unet_evaluate.py

```diff
@@ -0,0 +1,66 @@
+import tensorflow as tf
+from tqdm import tqdm
+
+from learning_wavelets.config import CHECKPOINTS_DIR
+from learning_wavelets.data.datasets import im_dataset_bsd68
+from learning_wavelets.evaluate import METRIC_FUNCS, Metrics 
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+
+
+
+def evaluate_unet(
+        noise_std_test=30,
+        run_id='ExactReconUnet_4_bsd500_0_55_None_1620201240',
+        n_epochs=500,
+        n_output_channels=1,
+        kernel_size=3,
+        base_n_filters=4, 
+        n_layers=4,
+        layers_n_non_lins=2,
+        non_linearity='relu',
+    ):
+    
+    noise_std_test = force_list(noise_std_test)
+    layers_n_channels = [base_n_filters*2**j for j in range(0,n_layers)]
```

Comment:
```suggestion
    layers_n_channels = [base_n_filters*2**j for j in range(0, n_layers)]
```

[Don't forget the space after the comma](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements).